.editorconfig

```diff
@@ -14,5 +14,5 @@ indent_style = tab
 indent_style = space
 indent_size = 2
 
-[{*.md}]
+[*.md]
```

Comment:
This rule didn't seem to be matching as PHPStorm kept removing the trailing whitespace. Also, the linter was giving the error:

![image](https://user-images.githubusercontent.com/16200219/98453941-af7c6000-2156-11eb-97f5-bfc7654bca87.png)

Once the braces were removed PHPStorm stopped removing the trailing whitespace.